usaspending_api/etl/helpers.py

```diff
@@ -141,3 +144,116 @@ def get_previous_submission(cgac_code, fiscal_year, fiscal_period):
         .order_by('-reporting_fiscal_period') \
         .first()
     return previous_submission
+
+
+def update_model_description_fields():
```

Comment:
Impressive!  Hopefully it will run in a tolerable amount of time, but I like being able to handle all those fields with a single general chunk of code.